modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/source/PollingSourceTestCase.java

```diff
@@ -51,6 +58,12 @@ protected String getConfigFile() {
     return "polling-source-config.xml";
   }
 
+  @Before
+  public void resetCounters() throws Exception {
+    PetFailingPollingSource.STARTED_POLLS = 0;
```

Comment:
This test is yielding false positives. The test pases because the number of invokations you want is correct, but you're not seeing that the instance that is actually modifying this is not the one you wanted. Change this by a Map<Source, Integer> and also assert which instances had how many invokations